credentials/java/com/google/auth/Credentials.java

```diff
@@ -43,6 +43,8 @@ public abstract class Credentials implements Serializable {
 
   private static final long serialVersionUID = 808575179767517313L;
 
+  public static final String GOOGLE_DEFAULT_UNIVERSE = "googleapis.com";
```

Comment:
should this be private?

Reply:
It's a different package than everything else, so we want to have it public to compare if current value is different etc